jmx/plugins/org.fusesource.ide.jmx.karaf/src/org/fusesource/ide/jmx/karaf/navigator/osgi/ServicesTableSheetPage.java

```diff
@@ -47,7 +46,7 @@ public void setView(IViewPage view) {
 	@Override
 	public void setActionBars(IActionBars actionBars) {
 		super.setActionBars(actionBars);
-		IMenuManager menu = actionBars.getMenuManager();
+		//IMenuManager menu = actionBars.getMenuManager();
```

Comment:
Effectively, bad code smell ^^
actionBars.getMenuManager() is also initilizating the field so I will keep the call but just remove the assignation